test/shell.mjs

```diff
@@ -43,12 +46,33 @@ const v4Object2 = `{
   ]
 }`;
 
+const registerHomedir = (container, subdir = "") => {
+  const __dirname = import.meta.dirname;
+  const homedir = path.join(__dirname, "../test/test-homedir", subdir);
+
+  container.register({
+    homedir: awilix.asFunction(() => homedir).scoped(),
```

Comment:
any reason to not register this as a static value? e.g.:
```
  homedir: awilix.asValue(homedir),
```

Reply:
Right. I feel better about donig this globally now, so I'll go do that.